README.md

```diff
@@ -69,6 +69,7 @@ Table of Contents
   * [Open Web Analytics](http://www.openwebanalytics.com/) - Google Analytics and Piwik alternative - `GPL` `PHP`
   * [Piwik](https://piwik.org/) - Leading open-source analytics platform that gives you more than just powerful analytics. - `GPLv3` `PHP`
   * [ThinkUp](https://www.thinkup.com/) - Insights into your social networking activity on Twitter, Facebook, Instagram, and beyond. ([Source code](https://github.com/ThinkUpLLC/ThinkUp)) - `GPL` `PHP`
+  * [Webalizer](http://www.webalizer.org/) - `GPL` `C`
```

Comment:
Please add a description; "fast, free web server log file analysis program" would be ok.